### ISSUE TITLE
Target module Dropout(p=0.05, inplace=False) is not supported. Currently, only the following modules are supported: `torch.nn.Linear`, `torch.nn.Embedding`, `torch.nn.Conv2d`, `transformers.pytorch_utils.Conv1D`.

### ISSUE THREAD

**Issue by aperalta1437** [huggingface/autotrain-advanced#853]
I fine-tuned a llama 3.1 8B model using the below command:

"autotrain llm --train --project-name some-project-name --model meta-llama/Llama-3.1-8B --data-path . --use-peft --lr 2e-4 --train-batch-size 5 --epochs 15 --trainer sft"

And it output the fine-tuned model which I am able to run (some-project-name). But, when I try to fine-tune on top of the model that was previously output (some-project-name) using below command, 

"autotrain llm --train --project-name some-project-name-2 --model ./some-project-name --data-path . --use-peft --lr 2e-4 --train-batch-size 5 --epochs 15 --trainer sft"

I see autotrain complaining about missing model_type which is suppossed to be located in a config.json file which should be at the root directory of the output model (some-project-name). So, to get over that issue, I copied the config.json file from the meta-llama/Llama-3.1-8B model located in HuggingFace repo and pasted out into the output model (some-project-name).

The previous issue was resolved but now autotrain complains about below issue:

Target module Dropout(p=0.05, inplace=False) is not supported. Currently, only the following modules are supported: `torch.nn.Linear`, `torch.nn.Embedding`, `torch.nn.Conv2d`, `transformers.pytorch_utils.Conv1D`.

I have no idea of how to fix it. I just know that the dropout "p" value that shows up above is located in the adapter_config.json file (lora_dropout property) which is located in the root directory of the model that was output (some-project-name).

I did the fine-tuning with transformers package version 4.48.2.

Can someone help?

I thought the process to fine-tune a model that was the output of another fine-tuning would be straight-forward.
